packages/ruleset/test/securityscheme-attributes.test.js

```diff
@@ -70,7 +70,7 @@ describe(`Spectral rule: ${ruleId}`, () => {
         expect(results).toHaveLength(1);
         expect(results[0].code).toBe(ruleId);
         expect(results[0].message).toBe(
-          `security scheme is missing required property: type`
+          `securityScheme is missing required property: type`
```

Comment:
Okay, I have one more potential wrench to throw in here 😅 

I'll first say, I have no problems with things as they are. I just noticed this and thought it was at least worth mentioning. If we decide this is too small a detail to worry about, I'm quite happy merging as is.

In another rule, you changed
> Operation with non-form requestBody

to be

> Operations with non-form request bodies

I'm wondering if we should be consistent here in using the topical name "Security scheme is missing..." vs the OpenAPI name "securityScheme" if we're doing that with other OpenAPI names elsewhere.

What do you think?

Reply:
When I made those changes to the securityscheme-attributes rule, I knew that would come back to bite me :)   Changed it back to "Security schema..."